Automattic/jetpack
# Publicize: Ensure that the auto-conversion setting is enabled by default

We want the auto-conversion setting (when available) to be enabled by default. This change updates the settings class so that the default value for the setting is based on whether the feature is available.

## Proposed changes:

* This updates the default value of the auto-conversion setting

### Other information:


## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
* Delete the `jetpack_social_settings` option e.g. `jetpack docker wp option delete jetpack_social_settings`
* Ensure the auto-conversion feature is enabled on your site
* Check the sharing settings in Jetpack or the admin screen in Jetpack Social.
* Without this change the auto-conversion setting will be disabled
* With this change the setting should be enabled
* Toggle the setting and then check the value with `jetpack docker wp option get jetpack_social_settings` It should reflect the toggle state
